### ISSUE TITLE
Render time synthetic data

### ISSUE THREAD

**Issue by hiralalkamil** [yoterel/STORM-Net#6]
I have managed to get the STORM-Net GUI running. However, when I try to render synthetic data, I don’t see any progress after 10-ish minutes of rendering, even though I did check the "Monitor Progress" box (I just see the green box moving from left to right).  The same goes for rendering the example model data.

When I close the GUI (while still trying to render the synthetic data), I do get the error “TypeError: cannot unpack non-iterable NoneType object”. With Traceback: “ File “main.py”, line 89, in <module> 
                                                                    sticker_locations, video_names = video.process_video(args)
                                                                                                                                                                                                                                                               
I was wondering if you have any clue what could be going on? Might this has something to do where all the files are stored or something? I guess the installation of all dependencies worked, since I managed to get the GUI started using Anaconda. 
Or does rendering synthetic data take a huge amount of time?

**Comment by yoterel** [comment]
Can you post here what operating system are you using? (name and version)
Also, did you download the correct renderer executable for your platform and pointed to it before rendering ?
Lastly, where are you placing the output? is it a folder which has permission to write into? (do any files appear there?)

Is it possible to ask that you try to render manually using the script? (notice instructions mention two ways of rendering, one is GUI and another is directly using render.py). Does this work for you ?

**Comment by yoterel** [comment]
closed as stale
if issue persists please reopen

**Comment by hiralalkamil** [comment]
I am sorry, wanted to reply to your last comment but it slipped from my mind. 

For future reference, using the script to render worked for me, but only after I placed the log and output folders on correct locations. I made the silly mistake of placing the template, output folder and log folder inside of the folder structure of STORM-Net, which did not work. Now that I placed the folders in a location the application has rights to write to, it all seems to work fine. @yoterel, thanks for your help!
